docs/tutorials/connecting-to-the-internet-via-wifi.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Setting up wifi on ev3
+subject: Networking
+---
+
+I like to setup wireless networking on my robots. Robots should go untethered! Here is how. It's easy in an interactive tool call `connmanctl`. You connect once, and next time you boot, it's all configured. On my ev3dev machine it went like this:
```

Comment:
Tutorials should be written formally, so we should try not to use the word "I". You can change the sentence to explain _why_ one might like it instead, without actually using the word "like".